Fix A-CLOSE support count

guptaanmol184 opened the pull request:
Fixes #5

guptaanmol184 committed:
Fix A-CLOSE support count

elixir-code reviewed (changes requested):
I suggest you add an additional boolean argument to 'AClose' function that allows us to toggle on and off returning support counts along with closed frequent item sets.

AClose(transactions, min_support, return_support_counts=False)

For Reference: function np.unique has a argument called 'return_counts' that allows us to optionally count the number of times each unique element appears in the array.

guptaanmol184 committed:
Add return_support_counts argument to AClose.

Toggles on and off returning support counts along with closed frequent item sets.

elixir-code commented:
Good work, @guptaanmol184 !
